pkg/types/layer.go

```diff
@@ -73,40 +72,6 @@ type Bom struct {
 	Packages []Package `yaml:"packages" json:"packages,omitempty"`
 }
 
-func validateDataAsBind(i interface{}) (map[interface{}]interface{}, error) {
-	bindMap, ok := i.(map[interface{}]interface{})
-	if !ok {
-		return nil, errors.Errorf("unable to cast into map[interface{}]interface{}: %T", i)
-	}
-
-	// validations
-	bindSource, ok := bindMap["Source"]
-	if !ok {
-		return nil, errors.Errorf("bind source missing: %v", i)
-	}
-
-	_, ok = bindSource.(string)
-	if !ok {
-		return nil, errors.Errorf("unknown bind source type, expected string: %T", i)
-	}
-
-	bindDest, ok := bindMap["Dest"]
-	if !ok {
-		return nil, errors.Errorf("bind dest missing: %v", i)
-	}
```

Comment:
I guess previously we always required 'Dest' in the dictionary form.
We did *not* require it in the simple string form; it would simply make dest = source.   I find 'source' without 'dest' wierd, I did make it work though.  I'm willing to drop that function. While weird, it does allow you to "not repeat yourself" in the event that you wanted to bind a path inside same as outside.

Reply:
well nothing depends on it now (or at least we did not have any tests prior to yesterday that tested it).
so we can drop it and not break any known behavior.

I'm feeling lazy, i guess just leave it in for now.  Point at this comment later when having done so causes problems to say "see, its your fault".